### PR TITLE
✨ Add FromQueryPackBundle, allowing conversion of an explorer bundle to a policy bundle.

### PR DESCRIPTION
**Pull request by preslavgerchev**
This is more of an internal feature, but a function to let us merge a cnquery (explorer) bundle and a policy bundle together.

### CODE REVIEW

File: policy/bundle.go [hunk 1]
```diff
@@ -1106,3 +1106,18 @@ func translateGroupUIDs(ownerMrn string, policyObj *Policy, uid2mrn map[string]s
 
 	return nil
 }
+
+// Takes a query pack bundle and does the following:
+// 1. Merges all queries from the two bundles together
+// 2. Merges all props from the two bundles together
+// 3. Puts all query packs in the policy bundle
+// 4. Convert all query packs to policies with a data only scoring system. Packs are still kept
+func (p *Bundle) MergeQueryPackBundle(bundle *explorer.Bundle) {
```
Comment (arlimus) [inline policy/bundle.go]: @preslavgerchev What if we changed this into a more generic `Merge(other *Bundle)` and instead have another `FromQueryPack(*explorer.Bundle)` function? It does require one more function, but they would be more specialized in their use-cases i.e. merge bundles together and convert any explorer.Bundle that we may receive. We currently have the `aggregateBundles` function in this file, which could be converted + exported into a `Merge(other *Bundle)` function. It also needs to combine the `Packs` field which is missing right now.

**Review by arlimus — CHANGES_REQUESTED** [review]
Super useful addition! I have a suggestion that splits the new call slightly and replaces an old internal-only call

File: policy/bundle.go (hunk 1 above)
Comment (preslavgerchev) [reply policy/bundle.go]: Sounds good!
Then I assume we could chain this into something like `bundle.Merge(FromQueryPack(explorerBundle))` where needed. I will work on that

**Comment by preslavgerchev** [comment]
@arlimus i've added the new `FromQueryPackBundle` func, this should be rfr again 🙌

**Review by arlimus — APPROVED** [review]
Awesome!! Thank you so much

**Pull request merged**